updater/packaging/debian/templates/Makefile

```diff
@@ -2,7 +2,7 @@ all: # nothing to build
 
 PKGNAME = hubble-enterprise
 INSTALLDIR = $(DESTDIR)/opt/autodesk/$(PKGNAME)
-SYSTEMDDIR = $(DESTDIR)/home/admin/.local/share/systemd/user
+SYSTEMDDIR = $(DESTDIR)/usr/lib/systemd/system
```

Comment:
Do you see a way to move our stuff into a subdirectory here? Would that make sense?

Reply:
I think that `systemd` doesn’t look for unit files in a subdirectory.